Formula/mask.rb

```diff
@@ -15,7 +15,7 @@ class Mask < Formula
   depends_on "rust" => :build
 
   def install
-    system "cargo", "install", *std_cargo_args
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "./mask"
```

Comment:
My repo directory structure changed today when I split my single crate into a bin and a lib. Cargo is failing because the root of the repo is now a Rust workspace, not the mask crate which is now a subdirectory.

Taking a guess on the fix here. Noticed that before #56869, this was the common pattern to use. However, I need to specify a relative path now for cargo to install the mask crate properly.